view_orgrepo_versions.js

```diff
@@ -0,0 +1,38 @@
+const Filehound = require('filehound')
+
+async function printGitURLs() {
+  const pathPromise = Filehound.create()
```

Comment:
no need for a separate var here, instead:
```
const pathList = await Filehound...
```

Reply:
No worries